geometries/intersection.json

```diff
@@ -0,0 +1,7 @@
+{
+	"id": 1108962113,
+	"name": "intersection",
```

Comment:
`intersection` is both a placetype and a prefix for a latitude/longitude value pair.. [example](https://raw.githubusercontent.com/whosonfirst-data/whosonfirst-data-intersection-us-ny/master/data/110/891/002/7/1108910027.geojson).

I think this json should detail the geometry (latitude/longitude), rather than the placetype.